projects/control-service/projects/helm_charts/pipelines-control-service/values.yaml

```diff
@@ -43,7 +43,7 @@ operationsUi:
     requests:
       memory: 250Mi
       cpu: 25m
-  config: '{"auth": {"skipAuth": true}}'
```

Comment:
So having an empty config is also a valid use case, correct? What I'm saying is, changing the empty config to a non-empty one -is not the bug fix.

Reply:
It's more like an immediate mitigation for the frontend not loading correctly issue. The actual problem is we don't get a default value if the `ignoreRoutes` config is missing altogether. I'll add that info to the bug ticket we have.

Reply:
Added info to https://jira.eng.vmware.com/browse/TAUR-6040